mod/state-transition/pkg/core/state_processor.go

```diff
@@ -437,82 +432,3 @@ func (sp *StateProcessor[
 	)
 	return st.SetLatestBlockHeader(lbh)
 }
-
-// getAttestationDeltas as defined in the Ethereum 2.0 specification.
-// https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#get_attestation_deltas
-//
-//nolint:lll
-func (sp *StateProcessor[
-	_, _, _, BeaconStateT, _, _, _, _, _, _, _, _, _, _, _, _, _,
-]) getAttestationDeltas(
-	st BeaconStateT,
-) ([]math.Gwei, []math.Gwei, error) {
-	// TODO: implement this function forreal
-	validators, err := st.GetValidators()
-	if err != nil {
-		return nil, nil, err
-	}
-	placeholder := make([]math.Gwei, len(validators))
-	return placeholder, placeholder, nil
-}
-
-// processRewardsAndPenalties as defined in the Ethereum 2.0 specification.
-// https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#process_rewards_and_penalties
-//
-//nolint:lll
-func (sp *StateProcessor[
-	_, _, _, BeaconStateT, _, _, _, _, _, _, _, _, _, _, _, _, _,
-]) processRewardsAndPenalties(
```

Comment:
Still used by the `examples/berad` module.

Provided by `examples/berad/pkg/state-transition/state_processor.go` row 476.

Called in `examples/berad/pkg/state-transition/state_processor.go` row 338.